### PR TITLE
Move to GitHub Actions

### PR DESCRIPTION
**Pull request by juanpicado**
Ref https://github.com/Level/community/issues/99

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![npm](https://img.shields.io/npm/v/level-test.svg?label=&logo=npm)](https://www.npmjs.com/package/level-test)
 [![Node version](https://img.shields.io/node/v/level-test.svg)](https://www.npmjs.com/package/level-test)
 [![Travis](https://img.shields.io/travis/com/Level/level-test.svg?logo=travis&label=)](https://travis-ci.com/Level/level-test)
```
Comment (vweevers) [inline README.md:8]:
```suggestion
[![Test](https://github.com/Level/level-test/actions/workflows/test.yml/badge.svg)](https://github.com/Level/level-test/actions/workflows/test.yml)
```

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
-[![Coverage Status](https://coveralls.io/repos/github/Level/level-test/badge.svg)](https://coveralls.io/github/Level/level-test)
+[![codecov](https://codecov.io/gh/Level/level-js/branch/master/graph/badge.svg)](https://codecov.io/gh/Level/level-test)
```
Comment (vweevers) [inline README.md:9]:
```suggestion
[![codecov](https://codecov.io/gh/Level/level-test/branch/master/graph/badge.svg)](https://codecov.io/gh/Level/level-test)
```

**Comment by vweevers** [comment]
Can you remove `.travis.yml`? Thank you!

**Comment by juanpicado** [comment]
:) ups ... I was almost blind the other day I forgot remove a lot of stuff. Thanks for the patches

**Review by vweevers — APPROVED** [review]
❤️

**Pull request merged**